frontend/src/employee-frontend/components/reports/AttendanceReservation.tsx

```diff
@@ -81,6 +81,13 @@ export default React.memo(function AttendanceReservation() {
     [unitId, filters]
   )
 
+  useEffect(() => {
+    const elements = document.getElementsByClassName('time-row-05:30')
+    if (elements && elements.length > 0) {
+      elements[0].scrollIntoView()
+    }
+  }, [report])
+
```

Comment:
`useRef` is better than finding an element by using a fixed class name.

Our own `scrollRefIntoView` is better than builtin `scrollIntoView`, because it does nothing in e2e tests, uses smooth scrolling, waits for the element to be in the dom, etc.

```
const autoScrollRef = useRef<HTMLTableRowElement>(null)
useEffect(() => {
  scrollRefIntoView(autoScrollRef)
}, [])

...

<Tr ref={autoScrollRef}>
```